modules/stanford_courses_importer/config/schema/stanford_courses_importer.schema.yml

```diff
@@ -0,0 +1,10 @@
+stanford_courses_importer.importer_settings:
```

Comment:
I'm guessing this was something left over from the H&S stuff? We shouldn't need this.

Reply:
removed